po/pl.po

```diff
@@ -315,44 +322,40 @@ msgstr ""
 "Wypełnij go źródłami RSS i spróbuj ponownie."
 
 #: src/controller.cpp:335
-#, fuzzy
 msgid ""
 "It looks like you haven't configured any feeds in your The Old Reader "
 "account. Please do so, and try again."
 msgstr ""
-"Wygląda na to, że nie skonfigurowałeś żadnych kanałów na swoim koncie Google "
-"Reader. Wykonaj to proszę i spróbuj ponownie."
+"Wygląda na to, że nie skonfigurowałeś żadnych kanałów na swoim koncie The "
+"Old Reader. Wykonaj to proszę i spróbuj ponownie."
 
 #: src/controller.cpp:340
-#, fuzzy
 msgid ""
 "It looks like you haven't configured any feeds in your Tiny Tiny RSS "
 "account. Please do so, and try again."
 msgstr ""
-"Wygląda na to, że nie skonfigurowałeś żadnych kanałów na swoim koncie Google "
-"Reader. Wykonaj to proszę i spróbuj ponownie."
+"Wygląda na to, że nie skonfigurowałeś żadnych kanałów na swoim koncie Tiny "
+"RSS. Wykonaj to proszę i spróbuj ponownie."
 
 #: src/controller.cpp:345
-#, fuzzy
 msgid ""
 "It looks like you haven't configured any feeds in your NewsBlur account. "
 "Please do so, and try again."
 msgstr ""
-"Wygląda na to, że nie skonfigurowałeś żadnych kanałów na swoim koncie Google "
-"Reader. Wykonaj to proszę i spróbuj ponownie."
+"Wygląda na to, że nie skonfigurowałeś żadnych kanałów na swoim koncie "
+"Newsblur. Wykonaj to proszę i spróbuj ponownie."
```

Comment:
NewsBlur like to capitalize "B" in their name, so please do so as well: https://www.newsblur.com/

Reply:
Done, BTW we have
```
msgid "ERROR: You must set `cookie-cache` to use Newsblur.\n"
```
and many translations keep it the same case.